app/mailers/application_mailer.rb

```diff
@@ -1,4 +1,4 @@
 class ApplicationMailer < ActionMailer::Base
-  default from: "workshop.portal@gmail.com"
+  default from: Rails.configuration.default_reply_to_email
```

Comment:
Tested it: You can still send emails, but the from-adress is still the correct one. So I guess gmail just ignores any from-fields that are not valid